website/docs/getting-started/setup/instance-configuration/authentication/google-login.md

```diff
@@ -7,64 +7,63 @@ sidebar_position: 1
 
 # Google OAuth
 
-Google OAuth is one of the most widely used custom authentication integrations. Appsmith allows you to integrate with Google for authenticating users to your apps.
+Appsmith allows you to integrate with Google to authenticate users to your apps.
 
-![Click to expand](</img/Google_Login.png>)
+## Configure Google Cloud
 
-To enable Google Sign in, login to your [google cloud console](https://console.cloud.google.com/)
+To enable Google Sign in, log in to your [google cloud console](https://console.cloud.google.com/) and follow the steps below:
 
```

Comment:
I think we should also tell them to create a new project and such.

Reply:
I think that's not the goal here. We want to tell the users how to configure OAuth. If a user comes to this page, he'll already know that he need to have a project.